mediapipe/examples/desktop/object_detection/README.md

```diff
@@ -24,15 +24,11 @@ python setup_ovms.py --get_models
 bazel build -c opt --define MEDIAPIPE_DISABLE_GPU=1 mediapipe/examples/desktop/object_detection:object_detection_ovms
 ```
 
-## Download the input video or prepare your own input as 'video.mp4'
```

Comment:
why remove this?

Reply:
this demo is using test video from /mediapipe/mediapipe/examples/desktop/object_detection/test_video.mp4